tests/src/test/scala/cats/collections/DietSpec.scala

```diff
@@ -101,6 +101,13 @@ class DietSpec extends CatsSuite {
     (Diet.empty[Int] - Range(10, 100)) should be (Diet.EmptyDiet())
   }
 
+  test("diet eq") {
```

Comment:
This demands at least one or two Scalacheck tests.

Reply:
I will add some of those